qmi/instruments/thorlabs/mpc320.py

```diff
@@ -0,0 +1,507 @@
+"""Module for a Thorlabs MPC320 motorised fibre polarisation controller."""
+
+from dataclasses import dataclass
+import logging
+from typing import Dict, List
+from qmi.core.context import QMI_Context
+from qmi.core.exceptions import QMI_InstrumentException
+from qmi.core.instrument import QMI_Instrument, QMI_InstrumentIdentification
+from qmi.core.rpc import rpc_method
+from qmi.core.transport import create_transport
+from qmi.instruments.thorlabs.apt_packets import (
+    HW_GET_INFO,
+    MOD_GET_CHANENABLESTATE,
+    MOT_GET_USTATUSUPDATE,
+    MOT_MOVE_ABSOLUTE,
+    MOT_MOVE_COMPLETED,
+    MOT_MOVE_HOMED,
+    MOT_SET_EEPROMPARAMS,
+    POL_GET_SET_PARAMS,
+)
+from qmi.instruments.thorlabs.apt_protocol import (
+    AptChannelJogDirection,
+    AptChannelState,
+    AptMessageId,
+    AptProtocol,
+)
+
+# Global variable holding the logger for this module.
+_logger = logging.getLogger(__name__)
+
+
+@dataclass
+class Thorlabs_MPC320_Status:
```

Comment:
Minor: For these (data)classes we do not apply the QMI-driver naming convention, so you could use the regular 'CamelCase' naming, without underscores.

Reply:
for this I want to start using the instrument as a prefix because then the client will know what its for. In general I want to start this in other QMI drivers as well. let me know what you think?

Reply:
I have no issue with that. The linter probably does complain about it though?

Reply:
linter seems happy with it